apis/v1alpha1/opentelemetrycollector_types.go

```diff
@@ -380,11 +380,13 @@ type OpenTelemetryTargetAllocatorPrometheusCR struct {
 	// PodMonitors to be selected for target discovery.
 	// This is a map of {key,value} pairs. Each {key,value} in the map is going to exactly match a label in a
 	// PodMonitor's meta labels. The requirements are ANDed.
+	// Empty or nil map matches all pod monitors.
```

Comment:
an explicit nil means select nothing iirc.

Reply:
It doesn't, there's no way to set nothing with these label maps. I think you're mixing this up with proper LabelSelectors in `v1alpha2`.

Reply:
@swiatekm-sumo so is it correct? Can it be merged or do I need to make any change?

Reply:
🤦 i am mixing those up.